src/screens/PutawayCandidates/index.tsx

```diff
@@ -21,13 +21,31 @@ class PutawayCandidates extends Component<Props> {
 
     this.state = {
       refreshing: false,
+      updatedlist: [],
     };
   }
 
   UNSAFE_componentWillMount() {
     this.getScreenData();
   }
 
+  componentDidUpdate() {
+    const {candidates} = this.props;
+    let updatedList: any = [];
+    if (candidates.length) {
+      this.props.candidates.map((list: any) => {
```

Comment:
You extracted candidates already from props. Instead of map use `filter`.
`const updatedList = candidates.filter(candidate => candidate.putawayStatus === 'READY');`